map.py

```diff
@@ -1,43 +1,55 @@
+#where the map lives.
 import time
 import game_commands
 import map_parser
+import monster_module
+import game_functions
+import game_parser
 
 TL = 9484
 BL = 9492
 TR = 9488
 BR = 9496
 HH = 9472
 VV = 9474
-EE = 32
+EE = 8901
 PL = 9791
+EN = None
 
 map_1 = [
         [TL, HH, HH, HH, HH, HH, HH, HH, HH, HH, HH, TR],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
-        [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
-        [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
-        [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
-        [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
+        [VV, EE, EE, EE, TL, HH, HH, TR, EE, EE, EE, VV],
+        [VV, EE, EE, EE, VV, EE, EE, EE, EE, EE, EE, VV],
+        [VV, EE, EE, EE, VV, EE, EE, VV, EE, EE, EE, VV],
+        [VV, EE, EE, EE, BL, HH, HH, BR, EE, EE, EE, VV],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
         [VV, EE, EE, EE, EE, EE, EE, EE, EE, EE, EE, VV],
         [BL, HH, HH, HH, HH, HH, HH, HH, HH, HH, HH, BR]
 ]
 
-user = {
-    "name": None,
-    "decision": None,
-    "column": 1,
-    "row": 1,
-    "symbol": chr(PL)
-}
+user = game_commands.player
```

Comment:
Leaving the concept of user in here is a bit leaky. It's probably more sensible to remove that concept entirely and create some functions that allow you to add things to the objects list and to set the user (if there's much that specifically needs the user). This way you don't have state laying across multiple files that needs to be synced. That will cause a lot of cruft over time.